extensions/data-workspace/src/dialogs/projectDashboard.ts

```diff
@@ -106,32 +106,86 @@ export class ProjectDashboard {
 	}
 
 	private createContainer(title: string, location: string): azdata.FlexContainer {
-		const dashboardData: IDashboardTable[] = this.projectProvider!.dashboardComponents;
-
 		const rootContainer = this.modelView!.modelBuilder.flexContainer().withLayout(
 			{
 				flexFlow: 'column',
 				width: '100%',
 				height: '100%'
 			}).component();
 
+		const headerContainer = this.createHeader(title, location);
+		const tableContainer = this.createTables();
+
+		rootContainer.addItem(headerContainer);
+		rootContainer.addItem(tableContainer);
+
+		return rootContainer;
+	}
+
+	/**
+	 * Create header with title, location and background
+	 */
+	private createHeader(title: string, location: string): azdata.Component {
+		const headerContainer = this.modelView!.modelBuilder.flexContainer().withLayout(
+			{
+				flexFlow: 'column',
+				width: '100%',
+				height: '30%'
+			}).component();
+
+		const header = this.modelView!.modelBuilder.flexContainer().withLayout(
+			{
+				flexFlow: 'column',
+				width: '100%',
+				height: '30%'
+			}).component();
+
 		const titleLabel = this.modelView!.modelBuilder.text()
 			.withProperties<azdata.TextComponentProperties>({ value: title, CSSStyles: { 'margin-block-start': '20px', 'margin-block-end': '0px' } })
 			.component();
-		rootContainer.addItem(titleLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'font-size': '36px', 'font-weight': '400' } });
+		header.addItem(titleLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'font-size': '36px', 'font-weight': '400' } });
 
 		const projectFolderPath = path.dirname(location);
 		const locationLabel = this.modelView!.modelBuilder.text()
 			.withProperties<azdata.TextComponentProperties>({ value: projectFolderPath, CSSStyles: { 'margin-block-start': '20px', 'margin-block-end': '0px' } })
 			.component();
-		rootContainer.addItem(locationLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'padding-bottom': '50px', 'font-size': '16px' } });
+		header.addItem(locationLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'padding-bottom': '50px', 'font-size': '16px' } });
+
+		const image = this.projectProvider!.image;
+		headerContainer.addItem(header, {
+			CSSStyles: {
+				'background-image': `url(${vscode.Uri.file(image!.light.toString())}),
+					linear-gradient(360deg, rgba(127,181,239,0.15) 0%, rgba(242,242,242,0) 42.2%, rgba(242,242,242,0.0572917) 42.2%)`,
```

Comment:
can we remove these colors, it will cause the text and background color's contrast ratio not meeting the requirement

Reply:
from the screenshot I can tell it is not optimized for dark theme

Reply:
Removed it for now. I will wait for the updated gradient as discussed.